src/Samples/TestEmbedded/TestEmbeddedLibrary/TestEmbeddedLibrary.csproj

```diff
@@ -11,6 +11,8 @@
     <CsWinRTWindowsMetadata>10.0.19041.0</CsWinRTWindowsMetadata>
     <SimulateCsWinRTNugetReference>true</SimulateCsWinRTNugetReference>
     <CsWinRTGenerateProjection>true</CsWinRTGenerateProjection>
+    <!-- Will expose enums as public in the projection
```

Comment:
Do we want a test or something for this option?

Reply:
I think it's fine to leave testing this particular case to manual testing. We'd need another library for the different projection, plus the bells and whistles. I think the impact of these changes is low enough to not warrant such additional testing infrastructure.